rkt/gc.go

```diff
@@ -207,19 +236,24 @@ func deletePod(p *pod) {
 		}
 		defer s.Close()
 
-		// execute stage1's GC
-		stage1TreeStoreID, err := p.getStage1TreeStoreID()
-		if err != nil {
-			stderr.PrintE("error getting stage1 treeStoreID", err)
-			stderr.Print("skipping stage1 GC")
-		} else {
+		if err := mountPodStage1(s, p); err == nil {
 			if globalFlags.Debug {
 				stage0.InitDebug()
 			}
-			stage1RootFS := s.GetTreeStoreRootFS(stage1TreeStoreID)
-			if err = stage0.GC(p.path(), p.uuid, stage1RootFS); err != nil {
+			if err = stage0.GC(p.path(), p.uuid); err != nil {
```

Comment:
it's confusing that we do `stage0.GC` but then have an error `performing stage1 GC` :|